### PR TITLE
Rename TestStep interface

### PR DESCRIPTION
**Pull request by haonliu**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Added relevant unit test for your changes. (`npm run test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`npm run precheckin`)
- [ ] Added screenshots/GIFs for UI changes.

#### Description of changes

(give an overview. Add technical description describing your changes)

#### Notes for reviewers

(put down any questions or special points for reviewers)

### CODE REVIEW

**Comment by haonliu** [comment]
> Hey @haonliu can you break this PR in smaller PR's? 91 files is a little hard to review.

I know it is hard to review... However, the renaming of 'TestStep' interface itself is being used in every requirement configuration file, that's why this PR is so big. and I cannot break that down any more.

File: src/assessments/types/requirement.ts [hunk 1]
```diff
@@ -55,6 +55,6 @@ export interface VisualHelperToggleConfig {
     assessmentNavState: AssessmentNavState;
     instancesMap: DictionaryStringTo<IGeneratedAssessmentInstance>;
     actionMessageCreator: DetailsViewActionMessageCreator;
-    isStepEnabled: boolean;
-    isStepScanned: boolean;
+    isRequirementEnabled: boolean;
```
Comment (dbjorge) [inline src/assessments/types/requirement.ts]: This is a breaking change to our persisted store data format. We can think about how to manage this, but it's a MUCH more frightening change than updating type and file names. Can you please revert the property rename from this PR to be considered separately?

File: src/assessments/assessment-builder.tsx [hunk 2]
```diff
@@ -239,12 +239,12 @@ export class AssessmentBuilder {
         } as Assessment;
     }
 
-    private static getStepConfig(steps: TestStep[], testStep: string): TestStep {
+    private static getStepConfig(steps: Requirement[], testStep: string): Requirement {
```
Comment (peterdur) [inline src/assessments/assessment-builder.tsx]: It's _so_ tempting to suggest renaming methods like `getStepConfig` and parameters like `steps`... but this PR is plenty big already, so it makes sense to just do the single mechanical rename now to keep the immediate change as simple as possible (91 files!) and defer additional renames to future PRs.

File: src/assessments/audio-video-only/test-steps/audio-only-equivalent.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { link } from '../../../content/link';
 import * as content from '../../../content/test/audio-video-only/audio-only-equivalent';
 import { ManualTestRecordYourResults } from '../../common/manual-test-record-your-results';
 import * as Markup from '../../markup';
-import { TestStep } from '../../types/test-step';
+import { Requirement } from '../../types/requirement';
 import { AudioVideoOnlyTestStep } from './test-steps';
```
Comment (peterdur) [inline src/assessments/audio-video-only/test-steps/audio-only-equivalent.tsx]: Similarly, a future separate PR should rename the test-steps folders to requirements, and the contained test-steps.ts files to requirements.ts.

File: src/assessments/types/requirement.ts (hunk 1 above)
Comment (haonliu) [reply src/assessments/types/requirement.ts]: Good point. But VisualHelperToggleConfig seems not being used in stores or in background. It is only used in views. Let me know if I'm misunderstanding.

File: src/assessments/assessment-builder.tsx (hunk 2 above)
Comment (haonliu) [reply src/assessments/assessment-builder.tsx]: I agree. there's a lot more place to change, we cannot fit them in a single PR.

File: src/assessments/types/requirement.ts (hunk 1 above)
Comment (dbjorge) [reply src/assessments/types/requirement.ts]: I'm sorry, you're right; I saw `isStepScanned` in the persisted data and got concerned, but it comes from the property by that name in `manual-test-status.ts`, not from here. Still, I'd rather keep PRs this big to single mechanical refactorings at a time

Comment (haonliu) [reply src/assessments/types/requirement.ts]: Cool, thanks.

**Pull request merged**